pyproject.toml

```diff
@@ -2,7 +2,7 @@
 requires = [
   "setuptools",
   "wheel",
-  "conan>=1.40.0",
+  "conan==1.59.0",
```

Comment:
Since I doubt we're going to fix 2.0.0 compatibility before the 0.12.0 release. I would make this and the other pins `<2.0.0` so users aren't tied to a particular release of conan to build from source.